test/package-managers/bun/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import chaiString from 'chai-string'
+import * as bun from '../../../src/package-managers/bun'
+
+chai.should()
+chai.use(chaiAsPromised)
+chai.use(chaiString)
+
+process.env.NCU_TESTS = 'true'
+
+describe('bun', function () {
+  it('list', async () => {
+    const versionObject = await bun.list({ cwd: __dirname })
```

Comment:
Though there are some old tests remaining that operate on test files in the project directory tree, I prefer a dynamic setup in a temp directory as seen here:

https://github.com/raineorshine/npm-check-updates/blob/c7a339c53a09caf1d6ed28f6e6e3c6e25eb26fec/test/bin.test.ts#L115-L128

It makes the tests slightly more verbose, but I've found it easier to manage by having everything related to a test contained in the test file. Plus cleanup is easier because you can just delete the temp directory rather than having to do a precise restore.